docs/docs/integrations/providers/opengauss.mdx

```diff
@@ -0,0 +1,149 @@
+# openGauss
```

Comment:
@mpb159753 I'll remove the provider page from the PR since it duplicates a lot of stuff with the vectorstore. You can PR a separate version of the provider page if you'd like after the fact